web/src/filter.js

```diff
@@ -1095,4 +1095,15 @@ export class Filter {
         }
         return true;
     }
+
+    is_conversation_view() {
+        if (
+            _.isEqual(this.sorted_term_types(), ["stream", "topic"]) ||
+            _.isEqual(this.sorted_term_types(), ["pm-with"]) ||
+            _.isEqual(this.sorted_term_types(), ["group-pm-with"])
+        ) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Could call `this.sorted_term_types()` and store as an variable, rather than repeatedly preforming the same operations over again.